addons/html_builder/static/src/core/building_blocks/builder_fontfamilypicker.xml

```diff
@@ -6,16 +6,16 @@
         <t t-foreach="fonts" t-as="font" t-key="font_index">
             <BuilderSelectItem t-props="forwardProps(font)">
                 <div class="d-flex justify-content-between">
-                    <span t-attf-style="font-family: {{font.styleFontFamily}};">
+                    <span t-attf-style="font-family: {{font.styleFontFamily}};" class="text-truncate">
                         <i t-if="font.type === 'cloud'" role="button" class="me-2 fa fa-cloud" title="This font is hosted and served to your visitors by Google servers"></i>
                         <t t-out="font.string"/>
                     </span>
-                    <div class="text-end o_select_item_only">
+                    <div class="text-end o_select_item_only ms-2">
                         <t t-if="font.indexForType >= 0">
                             <t t-set="delete_font_title">Delete this font</t>
                             <i role="button"
                                t-on-click.prevent.stop="() => this.onDeleteFontClick(font)"
-                               class="link-danger fa fa-trash-o o_we_delete_font_btn"
+                               class="fa fa-trash-o o_we_delete_font_btn"
```

Comment:
A detail but when we hover the trash icon, there is no hover effect anymore (it was already barely visible before so we can ignore it, but just so you know) 
Maybe the `link-dark` class could be used :thinking:

Reply:
@sobo-odoo I added a light red color for the hover state